guides/doc-Release_Notes/topics/katello-4.9.0.adoc

```diff
@@ -71,7 +71,7 @@ A full list of changes is available on https://projects.theforeman.org/issues?se
 * Package upgradable versions are not set correctly based on architecture - https://projects.theforeman.org/issues/36100[#36100]
 * Host cloning is broken - https://projects.theforeman.org/issues/36064[#36064]
 * Setting a Content Source is not persistent - https://projects.theforeman.org/issues/35834[#35834]
-* \'System purpose\' modal doesn\'t reset after switching hosts with switcher - https://projects.theforeman.org/issues/35324[#35324]
+* 'System purpose' modal doesn\'t reset after switching hosts with switcher - https://projects.theforeman.org/issues/35324[#35324]
```

Comment:
`doesn\'t` was rendered as `doesn't` in the page last time, so I left it alone

Reply:
Now you understand why my script generated wrong escaping: asciidoc escaping is hard.